src/pages/test.tsx

```diff
@@ -0,0 +1,89 @@
+// @ts-nocheck
+import React from "react";
+
+import AuthWall from "../auth/AuthWall";
+import {
+  addFacilityCapacity,
+  getMigrationFacilities,
+  getMigrationModelVersions,
+  getMigrationScenarios,
+} from "../database";
+import { totalIncarceratedPopulation } from "../impact-dashboard/EpidemicModelContext";
+import PageInfo from "../site-metadata/PageInfo";
+
+function calculateFacilityCapacity(modelInputs) {
+  const occupancyPct = modelInputs.facilityOccupancyPct;
+  const pop = totalIncarceratedPopulation(modelInputs);
+  if (pop && occupancyPct) {
+    return pop * occupancyPct;
+  }
+}
+
+// flip this to true to write changes to the database;
+// flip it to false for a dry run that just logs to console
+const write = false;
+
+const setFacilityCapacity = async ({
+  facilityDoc,
+  scenarioId,
+  modelVersionDoc,
+}) => {
+  let { modelInputs } = facilityDoc.data();
+  let modelVersionId;
+
+  if (modelVersionDoc) {
+    modelInputs = modelVersionDoc.data();
+    modelVersionId = modelVersionDoc.id;
+  }
+
+  const capacity = calculateFacilityCapacity(modelInputs);
+  if (capacity) {
+    addFacilityCapacity({
```

Comment:
It doesn't matter but it bothers me that the order is not the same as the order listed in the function signature itself. I know it doesn't matter.